addons/html_editor/static/src/main/toolbar/toolbar_plugin.js

```diff
@@ -1,6 +1,8 @@
 import { Plugin } from "@html_editor/plugin";
 import { reactive } from "@odoo/owl";
 import { Toolbar } from "./toolbar";
+import { ancestors, closestElement } from "../../utils/dom_traversal";
```

Comment:
To easily jump to the reference of imported functions, maybe
```suggestion
import { ancestors, closestElement } from "@html_editor/utils/dom_traversal";
```